src/lib.rs

```diff
@@ -463,6 +516,16 @@ mod tests {
         drain_logger(logger);
     }
 
+    fn test_count(logger: &mut Logger) {
+        let my_logger = ProgLogBuilder::new().unit(2).build();
+        for _i in 0..101 {
+            assert_eq!(my_logger.record_count(5), 2);
+        }
+        assert_eq!(my_logger.seen(), 505);
+        assert_eq!(logger.len(), 203);
```

Comment:
question: should this actually be 202? 

This will emit 2 log messages per 5 count. And the loop is for 101 iterations, which should be 202? I think that resolves the failing test.